components/org.wso2.carbon.micro.integrator.core/src/main/java/org/wso2/carbon/service/mgt/ServiceAdmin.java

```diff
@@ -110,7 +110,7 @@
     private static final String DATA_SERVICE_TYPE = "data_service";
 
 
-    public ServiceAdmin() throws Exception {
+    public ServiceAdmin() {
         super();
 //        pf = PersistenceFactory.getInstance(getAxisConfig());
 //        spm = pf.getServicePM();
```

Comment:
Shall we remove the commented out code?

Reply:
I didn't touch this code as I wasn't the one wrote them. I thought this still under development. But sure.